plugins/main/package.json

```diff
@@ -45,12 +45,15 @@
     "test:jest": "node scripts/jest --runInBand",
     "test:jest:runner": "node scripts/runner test",
     "generate:api-data": "node scripts/generate-api-data.js --spec https://raw.githubusercontent.com/wazuh/wazuh/$(node -e \"console.log(require('./package.json').version)\")/api/api/spec/spec.yaml --output file --output-directory common/api-info --display-configuration",
-    "prebuild": "node scripts/generate-build-version"
+    "prebuild": "node scripts/generate-build-version",
+    "knip": "npx knip --files"
```

Comment:
**suggestion**: the usage of `npx` in a package.json script should not be neccesary
```suggestion
    "knip": "knip --files"
```